extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -466,4 +477,17 @@ public String updateApiUrl(String original, String healthPath) {
     private static boolean shouldInclude(LaunchModeBuildItem launchMode, SmallRyeHealthConfig healthConfig) {
         return launchMode.getLaunchMode().isDevOrTest() || healthConfig.ui.alwaysInclude;
     }
+
+    private static String selectSchemeForManagement() {
+        Config config = ConfigProvider.getConfig();
```

Comment:
I would add a comment saying this is a best-effort approach. 
We also discussed a warning with @iocanel

Reply:
I understood that it should be INFO messages, not warnings. If so, these will be covered in a separate task (see the third point in epic https://github.com/quarkusio/quarkus/issues/33307).
I'll add a comment about the best-effort approach.